### PR TITLE
Add test for udevadm control -p

### PR DESCRIPTION
**Pull request by dtardon**
This is a follow-up for https://github.com/systemd/systemd/pull/29931.

### CODE REVIEW

File: test/units/testsuite-17.13.sh [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+# Set up a test device
+trap cleanup EXIT
+
+blk="$(mktemp)"
+truncate -s 1M "$blk"
+dev="$(losetup --show -f "$blk")"
+
+rules="/etc/udev/rules.d/99-test-17.13.rules"
```
Comment (yuwata) [inline test/units/testsuite-17.13.sh]: Better to use `/run`

File: test/units/testsuite-17.13.sh [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+    losetup -d "$dev"
+    rm -f "$blk" "$rules"
+}
+
+# Set up a test device
+trap cleanup EXIT
+
+blk="$(mktemp)"
+truncate -s 1M "$blk"
+dev="$(losetup --show -f "$blk")"
```
Comment (yuwata) [inline test/units/testsuite-17.13.sh]: Not a matter, but using /dev/null or so is simpler. See units/testsuite-17.04.sh

File: test/units/testsuite-17.13.sh [hunk 3]
```diff
@@ -0,0 +1,94 @@
...
+set -ex
+set -o pipefail
+
+# shellcheck source=test/units/util.sh
+. "$(dirname "$0")"/util.sh
+
+# Test for `udevadm control -p`
+
+test_not_property() {
+    [[ -z $(udevadm info --query property --property "$2" --value "$1") ]]
```
Comment (yuwata) [inline test/units/testsuite-17.13.sh]: `assert_eq "$(udevadm info ...)" ""` ?

File: test/units/testsuite-17.13.sh [hunk 4]
```diff
@@ -0,0 +1,94 @@
...
+}
+
+test_property() {
+    [[ $(udevadm info --query property --property "$2" --value "$1") = "$3" ]]
```
Comment (yuwata) [inline test/units/testsuite-17.13.sh]: `assert_eq "$(udevadm info ...)" "$3"` ?

**Review by yuwata — APPROVED** [review]

**Review by yuwata — APPROVED** [review]
LGTM. But please rebase.

**Review by bluca — APPROVED** [review]

**Pull request merged**